### PR TITLE
docs: add documentation about set up with Patchwork

### PR DESCRIPTION
**Pull request by rsarky**

### CODE REVIEW

**Comment by rralf** [comment]
Thanks, will test it on monday. We should take this patch to the wip-patchwork-integration branch.

**Comment by rsarky** [comment]
This PR is submitted to that branch itself. Do you want me to send you a patch instead?

File: README.md [hunk 1]
```diff
@@ -248,3 +248,40 @@ To compare all mails on the list against upstream:
...
+--------------------
+The results of PaStA's analyses can be used by [Patchwork](http://jk.ozlabs.org/projects/patchwork/).
+
+## Setting up PaStA and Patchwork
+Assuming a working setup of PaStA already exists, here are the steps necessary for Patchwork integration
+
+1. Install Patchwork on your system, following the guidelines in Patchwork's [documentation](https://patchwork.readthedocs.io/en/latest/development/installation/)
+
+2. Start a shell inside Patchwork's docker container with `docker-compose run --rm web --shell`
+
```
Comment (rralf) [inline README.md:262]: Could you please add two more words how to:
  - create a test project in patchwork
  - import parts of initial archive
  - import the rest of the archive

File: parse-pubin.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    sha = commit.split(' ')[0]
+    messages += [fromline + '\n' + subprocess.check_output(['git', '-C', gitdir, 'show', sha+':m'], encoding='UTF-8', errors='replace')]
+    # if i % 10000 == 0 and i != 0:
+    #     with open('publicinboxtmp', 'wb') as f:
+    #         f.write('\n'.join(messages).encode('utf-8'))
+        # subprocess.call(['python3', 'manage.py', 'parsearchive'] + parsemail_args + ['publicinboxtmp'])
+with open('publicinboxtmp', 'wb') as f:
+    f.write('\n'.join(messages).encode('utf-8'))
+subprocess.call(['python3', 'manage.py', 'parsearchive'] + parsemail_args + ['publicinboxtmp'])
```
Comment (rralf) [inline parse-pubin.py:38]: I bet this scripts works, but please, no, don't do it in this way. It's better to have a simple shell script. Just sent you an example per mail. :-)

File: README.md (hunk 1 above)
Comment (rsarky) [reply README.md:262]: Sure

File: parse-pubin.py (hunk 2 above)
Comment (rsarky) [reply parse-pubin.py:38]: Got it. This was just a hacky thing I set up using a script that Daniel wrote reference.
Will eventually clean this up and make it into a shells script

**Pull request closed without merging**